core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -52,6 +56,10 @@ oppia.factory('oppiaPlayerService', [
 
     var exploration = null;
 
+    var totalTimeStopwatch = null;
```

Comment:
"total time" is ambiguous -- total time with respect to what? It could be for a state or an exploration...

Maybe have an explorationStopwatch and a stateStopwatch?

Reply:
Done.